webhook/tests/test_main.py

```diff
@@ -0,0 +1,11 @@
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+
+from webhook.main import app
+
+client = TestClient(app)
+
+def test_index():
+    response = client.get('/')
+    assert response.status_code == 200
+    assert response.text == '{"data":"Hello World"}'
```

Comment:
You can do response.json() == {"msg": "Hello World"}

Reply:
Done